pkgs/os-specific/linux/libcap/default.nix

```diff
@@ -41,10 +43,12 @@ stdenv.mkDerivation rec {
       --replace 'man_prefix=$(prefix)' "man_prefix=$doc"
   '';
 
-  installFlags = [ "RAISE_SETFCAP=no" ];
+  patches = lib.optional static ./no-shared-lib.patch;
+
+  installFlags = [ "RAISE_SETFCAP=no" ] ++ makeFlags;
```

Comment:
`installPhase` already passes `makeFlags` to `make`:

https://github.com/NixOS/nixpkgs/blob/4b5c6c8a58a164faaa9347957081fae36c8dc474/pkgs/stdenv/generic/setup.sh#L1086-L1090